modules/javascript-udfs/partials/libraries-and-scopes.adoc

```diff
@@ -22,6 +22,7 @@ Furthermore, access restrictions can be applied to scopes, so that only certain
 
 include::partial$diagrams.adoc[tags=udf-scopes-diagram]
 
-The user does not call a JavaScript function (shown here as `getBusinessDays(startDate, endDate)`) directly; a *N1QL User-Defined Function* is defined to act a reference caller to the JavaScript function. 
+The user does not call a JavaScript function (shown here as `getBusinessDays(startDate, endDate)`) directly;
+an *{sqlpp} User-Defined Function* is defined to act a reference caller to the JavaScript function.
```

Comment:
Suggest leaving indefinite article unchanged (as above) for ease of conversion and consistency:

```suggestion
a *{sqlpp} User-Defined Function* is defined to act a reference caller to the JavaScript function.
```